drivers/lora/CMakeLists.txt

```diff
@@ -20,3 +20,6 @@ endif()
 
 zephyr_library_sources_ifdef(CONFIG_LORA_SX126X sx126x_standalone.c)
 zephyr_library_sources_ifdef(CONFIG_LORA_STM32WL_SUBGHZ_RADIO sx126x_stm32wl.c)
+zephyr_library_sources_ifdef(CONFIG_LORA_RYLRXXX rylrxxx.c)
+
+zephyr_library_include_directories_ifdef(CONFIG_LORA_RYLRXXX ${ZEPHYR_BASE}/drivers/modem)
```

Comment:
This shouldn't be required. If there is a header required externally, it should be under `${ZEPHYR_BASE}/include/drivers/modem`